### PR TITLE
website/integrations: added hoarder integration

### PR DESCRIPTION
**Pull request by shelldandy**
## Details

Added [Hoarder](https://hoarder.app/) Integration to docs.

### CODE REVIEW

**Review by dominic-r — COMMENTED** [review]
Just a few comments. Let me know if you have any questions.

**Comment by shelldandy** [comment]
@4d62 just addressed the comments and commited the suggestions, lmk if more tweaks are needed

**Review by dominic-r — COMMENTED** [review]
Hi, thanks for your response & corrections. It looks good, but I noticed a few more things that can be changed to make it perfect.

cc @tanberry for final review

**Comment by shelldandy** [comment]
@4d62 comments addressed lmk if more changes are needed

File: website/integrations/services/hoarder/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+> -- https://hoarder.app/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `hoarder.company` is the FQDN of the Hoarder install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## Authentik configuration
```
Comment (tanberry) [inline website/integrations/services/hoarder/index.md:23]:
```suggestion
## authentik configuration
```

File: website/integrations/services/hoarder/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+### Provider Settings
```
Comment (tanberry) [inline website/integrations/services/hoarder/index.md:25]:
```suggestion
### Provider settings
```

File: website/integrations/services/hoarder/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+In authentik, under **Applications** -> **Providers** of the **Admin interface**, create a new **OAuth2/OpenID Provider** with the desired settings.
+
+-   Name: `hoarder`
+-   Redirect URI: `https://hoarder.company/api/auth/callback/custom`
+
+Everything else is up to you, just make sure to grab the client ID and the client secret!
+
+### Application Settings
```
Comment (tanberry) [inline website/integrations/services/hoarder/index.md:34]:
```suggestion
### Application settings
```

**Comment by tanberry** [comment]
Thanks so much @shelldandy for this addition to our Integrations docs! (And thanks @4d62 for your edits, much appreciated.)

I made a few edits based on our title capitalization style; we use sentence case for titles, and the product name authentik is with a lower-case "a". Here's a link to our Style Guide for more info about titles and other things. https://docs.goauthentik.io/docs/developer-docs/docs/style-guide

Thanks again! Once you get those few edits made, let's merge!

**Comment by shelldandy** [comment]
@4d62 @tanberry comments addressed!

**Review by dominic-r — APPROVED** [review]
looks good to me. thanks for your contribution!

**Comment by shelldandy** [comment]
@tanberry can we merge now?

**Comment by tanberry** [comment]
> @tanberry can we merge now?

Hi @shelldandy thanks for your patience with this... the `authentik-ci-website / lint (prettier-check)` build check is still failing. I pulled to your PR locally, ran `make website`, but seem unable to push it back to your PR... I can get some help from the dev team tomorrow, or if you want to try to resolve this you can run these commands, then push again:

`make website-install` (run this to update your local build tools, we recently bumped up a version)
`make website`

**Review by tanberry — APPROVED** [review]
Thanks again for this contribution, @shelldandy !

**Pull request merged**